### DISCUSSION TITLE
Export in place of original file + disable "export is done" screen

### DISCUSSION THREAD

**Discussion by JinkoMinko** [mifi/lossless-cut#1931] [Q&A]
Hello!
I recently performed a clean windows install and lost all my Losslesscut settings. Now, I cannot manage to get it working like it used to:
- When exporting a video, I would like the original file to be replaced by the exported one, AKA trash or permanently delete the original and save the new one in the same folder. This way the file is already in the right place and I don't manually need to delete the original. Currently, the original gets trashed but I can only seem to set an export folder to which it will always export which is not ideal.
- Every time I export, I get an "export is done" pop up with general tips I don't care about and that I need to close before proceeding. Can I turn it off somehow?

Thanks!

**Comment by mifi** [comment]
> * When exporting a video, I would like the original file to be replaced by the exported one, AKA trash or permanently delete the original and save the new one in the same folder. This way the file is already in the right place and I don't manually need to delete the original. Currently, the original gets trashed but I can only seem to set an export folder to which it will always export which is not ideal.

I think you can use the "Delete source file" keyboard shortcut.

> * Every time I export, I get an "export is done" pop up with general tips I don't care about and that I need to close before proceeding. Can I turn it off somehow?

you can disable notifications under settings

**Reply by JinkoMinko** [reply]
Hi, thanks for the help!
> I think you can use the "Delete source file" keyboard shortcut.

I used to only click the export button and it would do it for me. I just checked my settings: 
**Cleanup files after export**
- ✅Trash auto-generated files
- ✅Trash original source file
- ✅Trash project LLC file
- ✅Permanently delete the files if trash fails
- Show this dialog every time
- ✅Do all of this automatically after exporting a file?

Yet the original file remains.
> You can disable notifications under settings

I had not noticed this option at the very bottom of the advanced options page, thanks.


You did not address my issue about the file exporting in the working directory rather than in the same folder as the source file. Any idea? I really can't find how to get this to work again. It was so convenient! I do remember using Losslesscut for many files in many different folders without ever needing to go to a working directory to fetch my files. I would just click the export button and that was it, the new exported file was right here in place of the original file.

**Comment by mifi** [comment]
> Yet the original file remains.

I just tested and it works for me. You're on Windows, right? I know there has been problems with deleting files on windows because windows tends to lock files. Do you see any warnings/errors in the developer tools? see here for how to open developer tools: https://github.com/mifi/lossless-cut/issues/new?assignees=&labels=&projects=&template=1-file-issue.yml

> * Currently, the original gets trashed

Aren't you here saying that the original **does** get deleted? In which cases does it not get deleted?

> You did not address my issue about the file exporting in the working directory rather than in the same folder as the source file. Any idea? I really can't find how to get this to work again. It was so convenient! I do remember using Losslesscut for many files in many different folders without ever needing to go to a working directory to fetch my files. I would just click the export button and that was it, the new exported file was right here in place of the original file.

Click the X next to the working dir

<img width="127" alt="Screenshot 2024-03-23 at 11 30 12" src="https://github.com/mifi/lossless-cut/assets/402547/2e9858cc-acd2-44ea-b52a-bcc932a56483">

**Reply by JinkoMinko** [reply]
I did get confused about the original getting deleted or not, sorry.
I have just unset my working directory and indeed it works as intended. And the original DOES get trashed. All my issues are fixed, thanks a lot!
